canonical/canonical.com
# fix: Microcloud link in the private cloud navigation

## Done

- Update the Microcloud link in the private cloud navigation
- Turn all microcloud links to relative

## QA

- Open the demo
- Navigate from Products > Private cloud > Microcloud
- See that you land on the Microcloud page
